src/Tizen.NUI.Components/Controls/Slider.cs

```diff
@@ -821,27 +819,27 @@ public override void ApplyStyle(ViewStyle viewStyle)
 
             if (null != sliderStyle?.Progress)
             {
-                CreateSlidedTrackAttributes();
+                CreateSlidedTrack();
```

Comment:
Not sure why u didn't attach Style postfix to function name.

Reply:
I checked CreateSlidedTrackAttributes, it doesn't related with style. So I removed style postfix.